web/src/routes/index.tsx

```diff
@@ -14,7 +14,7 @@ export default () => {
     <Router history={browserHistory}>
       <Switch>
         <Route path="/" exact component={Home} />
-        <Route path="/block" exact component={Block} />
+        <Route path="/block/:hash" exact component={Block} />
         <Route path="/blocklist/:pageNo?/:pageSize?" exact component={BlockList} />
```

Comment:
the paramters pass to router are a bit weired

Reply:
I will refactor later.